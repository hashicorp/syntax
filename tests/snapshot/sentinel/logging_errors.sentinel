# https://developer.hashicorp.com/sentinel/docs/language/logging-errors

value = 42
print("the value is", value) // the value is 42

map = { "foo": false }
print(map) // { "foo": false }

one_is_zero = rule { 1 == 0 }
print(one_is_zero) // false
