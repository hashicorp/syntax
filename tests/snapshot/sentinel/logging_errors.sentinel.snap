># https://developer.hashicorp.com/sentinel/docs/language/logging-errors
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.sentinel comment.line.number-sign.sentinel
>
>value = 42
#^^^^^^ source.sentinel
#      ^ source.sentinel keyword.operator.symbol.sentinel
#       ^ source.sentinel
#        ^^ source.sentinel constant.numeric.number.sentinel
>print("the value is", value) // the value is 42
#^^^^^ source.sentinel support.function.builtin.sentinel
#     ^ source.sentinel
#      ^ source.sentinel string.quoted.double.untitled
#       ^^^^^^^^^^^^ source.sentinel string.quoted.double.untitled
#                   ^ source.sentinel string.quoted.double.untitled
#                    ^^^^^^^^^ source.sentinel
#                             ^^^^^^^^^^^^^^^^^^ source.sentinel comment.line.double-slash.sentinel
>
>map = { "foo": false }
#^^^ source.sentinel keyword.control.sentinel
#   ^ source.sentinel
#    ^ source.sentinel keyword.operator.symbol.sentinel
#     ^^^ source.sentinel
#        ^ source.sentinel string.quoted.double.untitled
#         ^^^ source.sentinel string.quoted.double.untitled
#            ^ source.sentinel string.quoted.double.untitled
#             ^^ source.sentinel
#               ^^^^^ source.sentinel constant.language.sentinel
#                    ^^^ source.sentinel
>print(map) // { "foo": false }
#^^^^^ source.sentinel support.function.builtin.sentinel
#     ^ source.sentinel
#      ^^^ source.sentinel keyword.control.sentinel
#         ^^ source.sentinel
#           ^^^^^^^^^^^^^^^^^^^ source.sentinel comment.line.double-slash.sentinel
>
>one_is_zero = rule { 1 == 0 }
#^^^^^^^^^^^^ source.sentinel
#            ^ source.sentinel keyword.operator.symbol.sentinel
#             ^ source.sentinel
#              ^^^^ source.sentinel keyword.other.sentinel
#                  ^^^ source.sentinel
#                     ^ source.sentinel constant.numeric.number.sentinel
#                      ^ source.sentinel
#                       ^ source.sentinel keyword.operator.symbol.sentinel
#                        ^ source.sentinel keyword.operator.symbol.sentinel
#                         ^ source.sentinel
#                          ^ source.sentinel constant.numeric.number.sentinel
#                           ^^^ source.sentinel
>print(one_is_zero) // false
#^^^^^ source.sentinel support.function.builtin.sentinel
#     ^^^^^^^^^^^^^^ source.sentinel
#                   ^^^^^^^^ source.sentinel comment.line.double-slash.sentinel
>